packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -7,8 +7,8 @@ import { Link } from 'react-router-dom';
 
 const getStatusBadgeConfig = (
     mt5_account_status: TMT5AccountStatus,
-    openFailedVerificationModal?: (selected_account_type: string) => void,
-    selected_account_type?: string,
+    openFailedVerificationModal?: (selected_account_type: any) => void,
+    selected_account_type?: any,
```

Comment:
should be this type as any here?

Reply:
Yes. Because a proper mt5_login_list type isn't available. Initial type was provided as string. But that is incorrect. The function it is passed to is in tradershub store. And it looks like an object

Reply:
How bout giving a type of `<Record<string, unknown>>` will this type work? 🤔